hydrus/tests/test_parser.py

```diff
@@ -66,7 +66,7 @@ def test_get_data_from_location(self):
     def test_sanitise_path(self):
         """Test if the variables can be removed from the path"""
         path = "A/B/C/{id}"
-        result = openapi_parser.sanitise_path(path)
+        result = openapi_parser.sanitise_path(path
```

Comment:
Why isn't the function call complete?

Reply:
oh sorry , typo

Reply:
Also, the warning we talked about is still there.

Reply:
yeah i am trying to find out why that warning is coming , will fix it

Reply:
i think the warning is gone , which was because there was no autoincrement or default set